app/connectors/UserAnswersCacheConnector.scala

```diff
@@ -34,8 +34,8 @@ class UserAnswersCacheConnector @Inject()(
 
   private def url = s"${config.eventReportingUrl}/pension-scheme-event-reporting/user-answers"
 
-  def get(pstr: String, eventType: EventType)
-         (implicit ec: ExecutionContext, headerCarrier: HeaderCarrier): Future[Option[UserAnswers]] = {
+  private def getJson(pstr: String, eventType: EventType)
```

Comment:
I suspect that the 2 getJson methods here could be merged into one method since I think the only difference is the event type json field.